tapisproject_cms/Dockerfile

```
@@ -1,5 +1,5 @@
-# TACC/Core-CMS#v3.11.6
-FROM taccwma/core-cms:c5bea4c
+# TACC/Core-CMS#v4.5.0
+FROM taccwma/core-cms:5473db7
```

You may instead use:

```suggestion
FROM taccwma/core-cms:v4.8.0
```

_Only works since [v4.8.0](https://github.com/TACC/Core-CMS/releases/tag/v4.8.0)+._